### PR TITLE
Feature list Completion

### PR DESCRIPTION
**Pull request by RNtaate**
## What does this PR do?
- [X] A user can view all tasks and categories
- [X] A user can click on a single task to view all its details.
- [X] A user can create, edit,  and delete a category
- [X] A user  can create, edit,  and delete a task
- [X] Category and task data is stored using localStorage.

### CODE REVIEW

File: src/forms.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  } else {
+    low = 'checked';
+  }
+
+  const mySelectDiv = document.createElement('div');
+  const select = document.createElement('select');
+  select.setAttribute('id', 'todoCategories');
+  // if(existingTask != '') {
+  //   select.setAttribute('value', existingTask.getTaskCat());
+  // }
```
Comment (whiz25) [inline src/forms.js]: - remove this comment and any other comments in the code base

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
You have done an amazing job in this project. Your app is deployed online, OOP, Module patterns have been used, there is clear separation of concerns between the DOM and business logic.
Let's however, fix one thing before merging.
- [x] remove all comments from your code base


### Martin Afani

**Comment by RNtaate** [comment]
## Changes made
- [X] Removed all unwanted comments in the codebase.

**Review by sumancrest0001 — APPROVED** [review]
### Status: Approved :100: :1st_place_medal:

**Congratulations @RNtaate and @Rumbie-Mudzie7** 

Your project is approved as you made the change suggested by the previous TSE. Now, you can merge to the master and move ahead.

**All the best with the next project.
Happy coding !!!**

**Pull request merged**